src/inverseRobot.js

```diff
@@ -7,7 +7,12 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const newRobot = Object.fromEntries(Object.entries(robot)
+    .map(([key, value]) =>
+      [value, key]));
+
+  return Object.keys(newRobot).length === Object.values(robot).length
+    ? newRobot : null;
```

Comment:
The logic for checking repeated values is incorrect. This check will only work if the original object has repeated keys, not values. To correctly check for repeated values in the original object, you need to compare the size of the new set of values with the length of the original values array. If they are not equal, it means that there are repeated values in the original object.